Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -235,6 +234,7 @@ namespace SandboxEditor
 
     void ViewportManipulatorControllerInstance::UpdateViewport(const AzFramework::ViewportControllerUpdateEvent& event)
     {
+        m_usingMouseWrap = SandboxEditor::ManipulatorMouseWrap();
```

Comment:
Hmm I'm not sure I like this change sorry, I don't think we should need to call this every frame, I think we just want to better isolate the Settings Registry in the test (also I don't think we should move `m_virtualNormalizedPosition` outside the if block in the other commit. Let me try something locally with the settings reg to see if I can fix it without changing this code